src/components/media-player/media-player.js

```diff
@@ -0,0 +1,143 @@
+import React, { useEffect } from "react";
+import uniqueId from "lodash/uniqueId";
+
+import { VideoPlayer, AcceptOverlay } from "bootstrap-italia";
+import parse from "html-react-parser";
+import ReactMarkdown from "react-markdown";
+
+// TO DO
+// - [future] modular approach for local videos/youtubes/...
+// - [future] download thumbnail during build - eg. url: http://i3.ytimg.com/vi/_0j7ZQ67KtY/hqdefault.jpg
+
+function MediaPlayerEl({ url, lang, trascription, poster }) {
+  const messages = {
+    it: {
+      rememberLabel: "Ricorda per tutti i video",
+      acceptLabel: "Accetta",
+      trascriptionLabel: "Trascrizione",
+      cookiePolicy:
+        'Accetta i cookie di YouTube per vedere il video. Puoi gestire le preferenze nella <a href="/privacy-policy/#gestione-cookie" className="text-white">cookie policy</a>. ',
+    },
+    en: {
+      rememberLabel: "Remember my choice",
+      acceptLabel: "Accept",
+      trascriptionLabel: "Transcription",
+      cookiePolicy:
+        'You have to accept YouTube cookies to enable viewing of this video. You can manage the preferences via <a href="/privacy-policy/#gestione-cookie" className="text-white">cookie policy (italian language)</a>.',
+    },
+  };
+
+  let video = null;
+  const t = (key) => messages[lang][key];
+
+  const videoId = `video-js-${uniqueId("id_")}`;
+
+  useEffect(() => {
+    const script = document.createElement("script");
+    script.src =
+      "https://cdnjs.cloudflare.com/ajax/libs/videojs-youtube/3.0.1/Youtube.min.js";
+    document.head.appendChild(script);
+    // eslint-disable-next-line no-new
+    new AcceptOverlay(document.getElementById(`${videoId}-accept-video`), {
+      service: "youtube.com",
+    });
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    video = new VideoPlayer(document.getElementById(videoId));
```

Comment:
As soon as a page with a Youtube video is loaded, there are requests to youtube.com, even if the user didn't dismiss the overlay yet:

![image](https://github.com/italia/designers.italia.it/assets/788293/346ddeb7-4b5d-4c32-b402-671db9fcab45)

I think this is caused by this instantiation of the VideoPlayer, not sure though. If that's the case we should load if *after* the user's confirmation(?)

Reply:
WDYT @astagi ?

Reply:
It is true 🤔.
![screen](https://github.com/italia/designers.italia.it/assets/6736103/34e25a50-4685-4dc7-bc4e-c7586909d13c). 

Might be nice to load the script afterwards... and isolate the users browser from YouTube if they don't want to play the video. Is that possible? 

However I also confirm that there are no cookies set before consent as stated by the overlay message.

Reply:
Ok. Working on it. We choose to fix this in BSI loading those scripts only after "Accepted" the overlay.

Reply:
And we just integrated the Privacy policy here https://github.com/italia/designers.italia.it/pull/756/commits/8f4988e6498a885b9a0d6e49eacfa6336e75e20a